charts/kyverno/values.yaml

```diff
@@ -425,6 +425,10 @@ existingImagePullSecrets: []
 
 # Tests configuration
 test:
+  # enable or disable test pods
+  enabled: true
```

Comment:
do we need the `enabled` value ?

Reply:
@siddhikhapare ?

Reply:
> do we need the `enabled` value ?

I realized that I overlooked the ``enabled`` field in the [test configuration](https://github.com/kyverno/kyverno/blob/main/charts/kyverno/values.yaml#L427C1-L428C1), which should have been set to false. However, it was globally added under the templating settings in values.yaml which meant to disable the testing pod. I will update it.

Reply:
I mean we already have `--skip-tests` natively supported in helm 🤔

Reply:
> I mean we already have `--skip-tests` natively supported in helm 🤔

@eddycharly Could I only the update to increase the sleep duration in the chart? In chart sleep duration is 20 s is mentioned.

Reply:
Yup i think the duration changes you did are enough

Reply:
@siddhikhapare can you make the changes ?